cellprofiler/modules/reducenoise.py

```diff
@@ -37,20 +40,20 @@ def create_settings(self):
         super(ReduceNoise, self).create_settings()
 
         self.size = cellprofiler.setting.Integer(
-            doc="Size is the size of patches used for noise reduction.",
+            doc="Size of the patches to use for noise reduction.",
```

Comment:
Just for formatting consistency, do you mind moving the doc keywords here _below_ the text/value keywords?